src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOFalcon.kt

```diff
@@ -227,62 +220,86 @@ class SwerveModuleIOFalcon(
     kD: DerivativeGain<Velocity<Meter>, Volt>
   ) {
-    driveFalcon.config_kP(0, driveSensor.proportionalVelocityGainToRawUnits(kP))
-    driveFalcon.config_kI(0, driveSensor.integralVelocityGainToRawUnits(kI))
-    driveFalcon.config_kD(0, driveSensor.derivativeVelocityGainToRawUnits(kD))
+    val PIDConfig = Slot0Configs()
+
+    PIDConfig.kP = driveSensor.proportionalVelocityGainToRawUnits(kP)
+    PIDConfig.kI = driveSensor.integralVelocityGainToRawUnits(kI)
+    PIDConfig.kD = driveSensor.derivativeVelocityGainToRawUnits(kD)
+    PIDConfig.kV = 0.05425
+
+    driveFalcon.configurator.apply(PIDConfig)
   }
 
   override fun configureSteeringPID(
     kP: ProportionalGain<Radian, Volt>,
     kI: IntegralGain<Radian, Volt>,
     kD: DerivativeGain<Radian, Volt>
   ) {
-    steeringFalcon.config_kP(0, steeringSensor.proportionalPositionGainToRawUnits(kP))
-    steeringFalcon.config_kI(0, steeringSensor.integralPositionGainToRawUnits(kI))
-    steeringFalcon.config_kD(0, steeringSensor.derivativePositionGainToRawUnits(kD))
+    val PIDConfig = Slot0Configs()
+
+    PIDConfig.kP = steeringSensor.proportionalPositionGainToRawUnits(kP)
+    PIDConfig.kI = steeringSensor.integralPositionGainToRawUnits(kI)
+    PIDConfig.kD = steeringSensor.derivativePositionGainToRawUnits(kD)
+    PIDConfig.kV = 0.05425
+
+    driveFalcon.configurator.apply(PIDConfig)
   }
 
   override fun configureSteeringMotionMagic(
     maxVel: AngularVelocity,
     maxAccel: AngularAcceleration
   ) {
-    steeringConfiguration.motionCruiseVelocity = steeringSensor.velocityToRawUnits(maxVel)
-    steeringConfiguration.motionAcceleration = steeringSensor.accelerationToRawUnits(maxAccel)
+
+    val motionMagicConfig = MotionMagicConfigs()
+
+    motionMagicConfig.MotionMagicCruiseVelocity = steeringSensor.velocityToRawUnits(maxVel)
+    motionMagicConfig.MotionMagicAcceleration = steeringSensor.accelerationToRawUnits(maxAccel)
+
+    steeringFalcon.configurator.apply(motionMagicConfig)
   }
 
   override fun setDriveBrakeMode(brake: Boolean) {
+    val motorOutputConfig = MotorOutputConfigs()
+
     if (brake) {
-      driveFalcon.setNeutralMode(NeutralMode.Brake)
+      motorOutputConfig.NeutralMode = NeutralModeValue.Brake
     } else {
-      driveFalcon.setNeutralMode(NeutralMode.Coast)
+      motorOutputConfig.NeutralMode = NeutralModeValue.Coast
     }
+    driveFalcon.configurator.apply(motorOutputConfig)
   }
 
   override fun setSteeringBrakeMode(brake: Boolean) {
+    val motorOutputConfig = MotorOutputConfigs()
+
     if (brake) {
-      steeringFalcon.setNeutralMode(NeutralMode.Brake)
+      motorOutputConfig.NeutralMode = NeutralModeValue.Brake
     } else {
-      steeringFalcon.setNeutralMode(NeutralMode.Coast)
+      motorOutputConfig.NeutralMode = NeutralModeValue.Coast
     }
+    steeringFalcon.configurator.apply(motorOutputConfig)
+    // motor output configs might overwrite invert?
+    steeringFalcon.inverted = true
```

Comment:
motorOutputConfig also controls the invert so I added this incase it gets overrided from the previous line, even if it doesn't this couldn't hurt right?